ui/src/app/components/help-sidebar.tsx

```diff
@@ -672,18 +639,19 @@ export const HelpSidebar = fp.flow(
               </h3>,
             renderBody: () =>
               <HelpTips allowSearch={false}
```

Comment:
Looks like this shows the same content as the 'help' icon just without the search bar. Why do we need both on every page?

Reply:
Ah, this should always be showing notebooks help info actually. Thanks for the catch. Displaying it on every page is a change Lou asked for as we were thinking about how to further develop the workspace storage tab into something more interact with the user's actual google bucket.

Reply:
Ah, ok. Makes sense.